.github/workflows/update-ts-version-tags.yml

```diff
@@ -15,19 +15,24 @@ on:
     # https://crontab.guru/#5_8_*_*_1
     - cron: "5 8 * * 1"
 
+env:
+  CI: true
```

Comment:
Hm, why was this needed? GHA already should set this to `CI=1`.

Reply:
That's what I figured, but pnpm's guide had it and I copied it